nfs/inc/rpc_task.h

```diff
@@ -1671,6 +1699,14 @@ struct rpc_task
 
     enum fuse_opcode optype = (fuse_opcode) 0;
 
+    /*
+     * Proxy operation type.
+     * By default, this will be set to \p optype.
```

Comment:
Is there a value in setting it to optype?
I think defaulting it to 0 has an advantage that by checking this for non-zero we can tell if it's a proxy task.